meta-lmp-base/recipes-containers/docker/files/0001-daemon-overlay2-write-layer-metadata-atomically.patch

```diff
@@ -0,0 +1,62 @@
+From 4361790843d50f37d1e938b9bb90bd0d7d823aea Mon Sep 17 00:00:00 2001
+From: Mike Sul <mike.sul@foundries.io>
+Date: Wed, 13 Sep 2023 09:13:13 +0200
+Subject: [PATCH] daemon: overlay2: Write layer metadata atomically
+
+When the daemon process or the host running it is abruptly terminated,
+the layer metadata file can become inconsistent on the file system.
+Specifically, `link` and `lower` files may exist but be empty, leading
+to overlay mounting errors during layer extraction, such as:
+"failed to register layer: error creating overlay mount to <path>:
+too many levels of symbolic links."
+
+This commit introduces the use of `AtomicWriteFile` to ensure that the
+layer metadata files contain correct data when they exist on the file system.
+
```

Comment:
can please add the patch status?
```
Upstream-Status: Backport [https://github.com/moby/moby/pull/46471]
```
this will help droping the patch on next update.

Reply:
Done.